nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/chronicle/ChronicleLogTailer.java

```diff
@@ -142,14 +143,19 @@ public void toEnd() {
     public void toStart() {
         log.debug(String.format("toStart: %s", id));
         cqTailer.toStart();
+        if (! cqTailer.state().equals(TailerState.FOUND_CYCLE)) {
+            log.warn("Unable to move to start because the tailer is not initialized, " + this);
+        }
     }
 
     @Override
     public void toLastCommitted() {
         long offset = offsetTracker.getLastCommittedOffset();
         if (offset > 0) {
             log.debug(String.format("toLastCommitted: %s, found: %d", id, offset));
-            cqTailer.moveToIndex(offset);
+            if (! cqTailer.moveToIndex(offset)) {
+                log.warn("Unable to move to the last committed offset, " + this);
```

Comment:
Is this important? Should we throw instead? Or just make this a debug message if it's not important? (same for seek)